gradle/testing/randomization/policies/solr-tests.policy

```diff
@@ -56,6 +56,15 @@ grant {
 
   // needed by randomizedtesting runner to identify test methods.
   permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
+  permission java.lang.reflect.ReflectPermission "newProxyInPackage.dev.langchain4j.model.cohere";
+  permission java.lang.reflect.ReflectPermission "newProxyInPackage.dev.ai4j.openai4j";
+  permission java.lang.reflect.ReflectPermission "newProxyInPackage.dev.langchain4j.model.huggingface";
+  permission java.net.SocketPermission "api.cohere.ai", "accept,listen,connect,resolve";
+  permission java.net.SocketPermission "api.openai.com", "accept,listen,connect,resolve";
+  permission java.net.SocketPermission "api.mistral.ai", "accept,listen,connect,resolve";
+  permission java.net.SocketPermission "api-inference.huggingface.co", "accept,listen,connect,resolve";
+  permission java.net.SocketPermission "langchain4j.dev", "accept,listen,connect,resolve";
```

Comment:
Do our tests need to communicate with these services over the internet?!

Reply:
No, it was there for some on the fly checks I did before refining the tests with the dummy model, I'm cleaning up this with the next commit